In [8]:
import numpy as np
import matplotlib as plt


# Rules:

In [9]:
def rule1(dA,c):
    return c*dA

In [10]:
def rule2(A,dA,m,c):
    return c*m*(A**(m-1))*dA

In [11]:
def rule3(dA,dB):
    dQ = np.sqrt(dA**2 + dB**2)
    return dQ

In [13]:
def rule4(Q,A,dA,m,B,dB,n):
    s = ((m*dA/A)**2)+((n*dB/B)**2)
    return Q*np.sqrt(s)

### $F\_c=\$ 

In [16]:
#from last week's data

mass = .2083
d_mass = .0001
F = 0.7888683507240091
slope = 10.424241737471668
d_slope = 1.1929392256645142

rule4(F,mass,d_mass,1,slope,d_slope,1)

0.09027805607544004

In [17]:
#The results from last week is the same as this one, as I also used python to calculate the error

In [24]:
#Continue in your Jupyter Notebook
#1. For the following dataset, calculate the mean (average) and the error in the mean using the numpy functions
#of np.std() for standard deviation and np.average(). x = 1.1, 1.3, 1.4, 0.9, 0.95, 1.05
#2. Write out your results in a sentence.

x = 1.1, 1.3, 1.4, 0.9, 0.95, 1.05
st = np.std(x)
avg = np.average(x)
error = st/np.sqrt(len(x))
print(error)

0.073282810879294


In [23]:
print(st)
print(avg)

0.17950549357115014
1.1166666666666667


For the dataset of x the standard deviation $\sigma=.18$ and the average = $1.116\pm.07$

### Rule3

$\delta Q = Q\sqrt{(\delta A)^2+(\delta B)^2}$


### $F_c$

$\delta F_c = F_c \sqrt{(1\frac{\delta m}{m})^2 + 1\frac{\delta slope}{slope})^2 }$